interviewQuestions/Minimum sum partition

```diff
@@ -0,0 +1,64 @@
+// Partition a set into two subsets such that the difference of subset sums is minimum
```

Comment:
@swatiprajapati08 make comment style like this and refactored the code according to Google Java Style Guide
```java
/**
 * Partition a set into two subsets such that the difference of subset sums is minimum.
 **/
```